core/core_bind.h

```diff
@@ -423,6 +423,7 @@ class ClassDB : public Object {
 
 protected:
 	static void _bind_methods();
+	static void _bind_compatibility_methods();
```

Comment:
```suggestion
#ifndef DISABLE_DEPRECATED
	Dictionary class_get_signal_compat_82198(StringName p_class, StringName p_signal) const;
	TypedArray<Dictionary> class_get_signal_list_compat_82198(StringName p_class, bool p_no_inheritance = false) const;
	TypedArray<Dictionary> class_get_property_list_compat_82198(StringName p_class, bool p_no_inheritance = false) const;
	TypedArray<Dictionary> class_get_method_list_compat_82198(StringName p_class, bool p_no_inheritance = false) const;
#endif
	static void _bind_compatibility_methods();
```
These should be kept together